Automattic/wp-calypso
# Replace onClick callbacks by links on ActivityCard + fix copies and styles

#### Changes proposed in this Pull Request
**Very small PR**

Replace all the `onClick` callbacks by links in the `ActivityCard` component. This will resolve an issue with navigation and its history, we can't go back to the previous page when we click on the Restore to this point" and "Download backup" buttons in the `ActivityCard` component

Also, apply some styles to match the design:

- Applied the new button styling (without border and color)
- Applied the right text color
- Added breakpoints to adapt the font size to the desktop version

Before styles applied:

![image](https://user-images.githubusercontent.com/9832440/79476036-6b7e0e00-8000-11ea-930a-3a4db3bcf7e6.png)

After:
![image](https://user-images.githubusercontent.com/9832440/79476112-7cc71a80-8000-11ea-9217-d0052f4cf0ab.png)

#### Testing instructions

- Apply the change and with a site with a Real-time Backup subscription, go to a date that has activities rewindable.
- Click on "Actions" in the Activity Card, and click on the "Restore to this point" and "Download backup" buttons and see that they work as expected. Then go back (with the back button of your browser) to the Backup Status page and you will be on the backup date that you selected before.
